.zenodo.json

```diff
@@ -0,0 +1,6 @@
+{
+   "license": {
+      "id": "Apache-2.0"
+   }, 
+   "title": "Guide to make C++ available as a web application"
```

Comment:
```suggestion
   "title": "Guide to convert C++ into a web application"
```

Reply:
Title is OK to me as it is, you are right we are converting c++ 2 wasm, but the cli, cgi and Python are running the C++ as is. To me 'make C++ available` covers those cases aswell.